orca-core/src/main/groovy/com/netflix/spinnaker/orca/pipeline/model/Stage.java

```diff
@@ -67,6 +67,10 @@ public Stage(T execution, String type) {
    */
   private String name;
 
+  public String getName() {
+    return name != null ? name : type;
```

Comment:
I really want to add `@Nullable` / `@NonNull` annotations on these model classes but it makes Lombok change the constructors so I need to make a larger change.